common/app/facebookimages/openGraphOverlay.scala

```diff
@@ -0,0 +1,20 @@
+package facebookimages
+
+import model.Article
+import views.support.{FacebookOpenGraphImage, ImgSrc}
+
+
+object openGraphOverlay {
```

Comment:
Why not making this object something like `ArticleWithOpenGraphOverlay` with an `apply` method that takes an article as argument?